theme/js/main.js

```diff
@@ -1,11 +1,8 @@
-const {message} = require("./_module");
-
-console.log(message);
-
 // Mobile Menu
 const mobMenu = document.querySelector("#mobile-nav");
 mobMenu.addEventListener("change", e =>
 {
-  const data = document.querySelector(".data");
-  data.textContent = `Data to be added here.`;
+    const optionTitle = e.target.selectedOptions[0].label.split(' ').join('');
```

Comment:
I am aware of linting errors. I forgot to pass the tests. Let's leave it for the next commit.